### PR TITLE
Updated third party notice and wording in docs.

### PR DESCRIPTION
**Pull request by monikavar**
add third party notice separately and wording in docs

### CODE REVIEW

File: docs/modules/publisher-migrationpath.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [Home](../../readme.md)
 
-# Migration path for publisher version 2.5.x to 2.8.2
+# Migration path for OPC Publisher v2.5.x to v2.8.2
```
Comment (karok2m) [inline docs/modules/publisher-migrationpath.md]: Let's not use `v` in front of OPC Publisher version numbers. Our tags are without any prefixes.

Comment (monikavar) [reply docs/modules/publisher-migrationpath.md]: I was using for version short form only and not for tag names. Let me rephrase it.

**Review by karok2m — APPROVED** [review]
